stile/hsc/base_tasks.py

```diff
@@ -832,17 +857,35 @@ def _makeArgumentParser(cls):
 class PatchSingleEpochStileConfig(CCDSingleEpochStileConfig):
     sys_tests = adapter_registry.makeField("tests to run", multi=True,
                     default=["StatsPSFFlux", #"GalaxyXGalaxyShear", "BrightStarShear",
-                             "StarXGalaxyShear", "StarXStarShear",
-                             #"WhiskerPlotStar", "WhiskerPlotPSF", "WhiskerPlotResidual",
-                             #"ScatterPlotStarVsPSFG1", "ScatterPlotStarVsPSFG2",
-                             #"ScatterPlotStarVsPSFSigma", "ScatterPlotResidualVsPSFG1",
-                             #"ScatterPlotResidualVsPSFG2", "ScatterPlotResidualVsPSFSigma"
+                             "StarXGalaxyShear", "StarXStarShear", "Rho1",
+                             "WhiskerPlotStar", "WhiskerPlotPSF", "WhiskerPlotResidual",
+                             "ScatterPlotStarVsPSFG1", "ScatterPlotStarVsPSFG2",
+                             "ScatterPlotStarVsPSFSigma", "ScatterPlotResidualVsPSFG1",
+                             "ScatterPlotResidualVsPSFG2", "ScatterPlotResidualVsPSFSigma"
                              ])
+    whiskerplot_figsize = lsst.pex.config.ListField(dtype=float,
+        doc="figure size for whisker plot", default = [7., 5.])
+    whiskerplot_xlim = lsst.pex.config.ListField(dtype=float,
+        doc="x limit for whisker plot", default = [None, None])
+    whiskerplot_ylim = lsst.pex.config.ListField(dtype=float,
+        doc="y limit for whisker plot", default = [None, None])
+    whiskerplot_scale = lsst.pex.config.Field(dtype=float,
+        doc="length of whisker per inch", default = 0.4)
     coaddName = lsst.pex.config.Field(
         doc = "coadd name: typically one of deep or goodSeeing",
         dtype = str,
         default = "deep",
     )
+    # Generate a list of flag columns to be used in the .removeFlaggedObjects() method
```

Comment:
Is there a reason to add a copy of this here?  It should just inherit directly from `CCDSingleEpochStileConfig`.